scripts/start.js

```diff
@@ -1,12 +1,10 @@
-const static = require('node-static')
-const { createServer } = require('http')
+const serve = require("node-static");
```

Comment:
Wow, I used a reserved word as an identifier and it didn't throw? That's surprising. Obviously this is a good change.

Who caught this and made the change? You or `prettier`?